### DISCUSSION TITLE
Julia

### DISCUSSION THREAD

**Discussion by Anzo52** [codinasion/codinasion-programme#896] [Ideas]
So, I just watched ![this video](https://youtu.be/JYs_94znYy0) (fireship is awesome), and Julia looks like an interesting language. I don't know it but I'm gonna look into it more. What do you think about adding it to the list of languages we're using codinasion?

**Comment by harshraj8843** [comment]
Hey @Anzo52 

Thanks for suggestion 👍🏻 

`Julia` has already been considered for addition. [See here...](https://github.com/codinasion/codinasion-programme/issues/559#issuecomment-1158144830)

@SanjayShetty01 has [raised a PR to add Julia programmes](https://github.com/codinasion/codinasion-programme/pull/885) which will be merged shortly after review
